README.md

```diff
@@ -44,30 +40,39 @@ Writing tests is simply a case of implementing the types and registering them wi
 package main
 
 import (
-    "github.com/form3tech-oss/f1/v2/pkg/f1"
-    "github.com/form3tech-oss/f1/v2/pkg/f1/testing"
-    "fmt"
+	"fmt"
+	
+	"github.com/form3tech-oss/f1/v2/pkg/f1"
+	"github.com/form3tech-oss/f1/v2/pkg/f1/testing"
 )
 
 func main() {
-    testing.Add("mySuperFastLoadTest", setupMySuperFastLoadTest)
-    f1.Execute()
+	f1.Scenarios().Add("mySuperFastLoadTest", setupMySuperFastLoadTest).Execute()
```

Comment:
Still not fully convinced by this naming seeing it written here - `Scenarios` is constructing something, so should it be a `New` function?

```suggestion
	f1.New().Add("mySuperFastLoadTest", setupMySuperFastLoadTest).Execute()
	f1.NewBuilder().Add("mySuperFastLoadTest", setupMySuperFastLoadTest).BuildAndRun()
```

Reply:
Not a big deal though if we have lots of dependencies already

Reply:
Sounds good to me. Probably is better to change it now than later.